docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -0,0 +1,94 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-dfanalytics-resources]]
+=== {dfanalytics-cap} resources
+
+A {dfanalytics} configuration object has the following properties:
+
+`analysis`::
+  (object) The type of analysis that is performed on the `source`. For example: 
+  `outlier_detection`. For more information, see <<dfanalytics-types>>.
+  
+`analyzed_fields`::
+  (object) You can specify both `includes` and/or `excludes` patterns. If 
+  `analyzed_fields` is not set, only the relevant fileds will be included. For 
+  example all the numeric fields for {oldetection}.
+
+`dest`::
+  (object) The destination configuration of the analysis. For more information, 
+  see <<dfanalytics-dest-resources>>.
+
+`id`::
+  (string) The unique identifier for the {dfanalytics-job}. This identifier can 
+  contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and 
+  underscores. It must start and end with alphanumeric characters. This property 
+  is informational; you cannot change the identifier for existing jobs.
+  
+`model_memory_limit`::
+  (string) The approximate maximum amount of memory resources that are 
+  required for analytical processing. The default value for {dfanalytics-jobs} 
```

Comment:
I'm not sure this first sentence reads quite right. I'd say more "The approximate maximum amount of memory resources that are permitted for analytical processing." We won't use this much if the analysis requires less for example.

Reply:
Also, do we need to say anything about the behaviour when the analysis will exceed the memory limit(?), i.e. add a sentence along the lines of: "If the analysis requires more than `model_memory_limit` then it will not run and report how much memory it requires to run."